db/migrate/20160625192919_create_configurations.rb

```diff
@@ -1,6 +1,6 @@
-class CreateConfigs < ActiveRecord::Migration
+class CreateConfigurations < ActiveRecord::Migration
   def change
-    create_table :configs do |t|
+    create_table :configurations do |t|
     	t.integer :possible_min
     	t.integer :possible_max
 
```

Comment:
Thanks for doing this, but it's not really necessary. I'd like to avoid having to re-run migrations on existing databases. This tool has already been running for a few weeks, so someone will have to rename the table through the Heroku console. Since this project is somewhat "in production" already, it's better to write a new migration that renames the table.